dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -278,7 +403,16 @@ function get_free_disk_bytes {
 }
 
 function get_partition_table_type {
-    blkid -s PTTYPE -o value "$1"
+    case $(basename "$1") in
+        # Assume DASD if the device starts with "dasd" blkid does not
+        # properly recognize dasd as PTTYPE bsc#1209247
+        dasd*)
```

Comment:
Uhm, this really smells. There is no programatical way to detect the partition table type on a DASD ?

Reply:
```
k3s01:~ # parted /dev/dasda print
Model: IBM S390 DASD drive (dasd)
Disk /dev/dasda: 32.2GB
Sector size (logical/physical): 512B/4096B
Partition Table: dasd
Disk Flags: 

Number  Start   End     Size    File system  Flags
 1      98.3kB  315MB   315MB   ext2
 2      315MB   32.2GB  31.9GB  btrfs
```

Reply:
imho if we have to keep this, the code should be selected exclusively by architecture. DASD only exists on s390/s390x.